tests/test-debian.sh

```diff
@@ -6,7 +6,8 @@ cat <<EOF | docker build --force-rm -t githooks:debian-base -
 FROM debian
 RUN apt-get update && \
     apt-get install --no-install-recommends --no-install-suggests -y \
-        git-core
+        git-core apt-transport-https ca-certificates && \
+    update-ca-certificates
```

Comment:
Curious why was this necessary now?

Reply:
Ehm, huch, I already forgot. I was necessary because it couldn't connect to `github.com` because of certificates errors...
This fixed it, hm... better solution?